src/frontend/admin/src/hooks/useResources/useResourcesRoot.ts

```diff
@@ -101,7 +102,7 @@ export const useResourcesRoot = <
 
   const actualMessages = useMemo(
     () => ({ ...messages, ...resourceMessages }),
-    [messages, resourceMessages]
+    [resourceMessages]
```

Comment:
Do not understand why you removed `messages` from hook dependencies

Reply:
Because it refers to the messages constant of the default translations